core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci

```diff
@@ -61,3 +61,5 @@ gcr.io/istio-testing/build-tools:release-1.14-204316d1ee4013432e0b35d8ceba3a943e
 quay.io/maistra-dev/maistra-proxy-builder:2.0 registry.ci.openshift.org/ci/maistra-proxy-builder:2.0
 quay.io/maistra-dev/maistra-proxy-builder:2.1 registry.ci.openshift.org/ci/maistra-proxy-builder:2.1
 quay.io/almartin/odf-console-ci-runner:latest registry.ci.openshift.org/ci/odf-console-ci-runner:latest
+quay.io/strimzi-qe/strimzi-tests:release-0.29.x registry.ci.openshift.org/ci/strimzi-tests:release-0.29.x
```

Comment:
The changes to this file will need to be merged so that the images can be propagated across the build clusters before we can run a rehearsal job using this image.

Reply:
Opened https://github.com/openshift/release/pull/37512